index.html

```diff
@@ -18,13 +18,14 @@
                 </div>
                 <div class="nav__links">
                     <ul class="nav__links-items">
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="# ">Contact</a></li>
+                        <li><a href="#portfolio">Portfolio</a></li>
+                        <li><a href="#about">About</a></li>
+                        <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Check the image 
![image](https://user-images.githubusercontent.com/62705947/215529398-3d06c327-bb74-4c78-b0c5-ea87f1c8cfeb.png)

Reply:
- [ ] Remove the descriptions from your readme file 
![image](https://user-images.githubusercontent.com/62705947/215530617-95547962-310a-4607-a78d-0558ed6daf67.png)